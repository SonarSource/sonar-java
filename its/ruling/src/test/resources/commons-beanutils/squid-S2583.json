{
'commons-beanutils:commons-beanutils:src/test/java/org/apache/commons/beanutils/bugs/Jira347TestCase.java':[
112,
],
'commons-beanutils:commons-beanutils:src/test/java/org/apache/commons/beanutils/memoryleaktests/MemoryLeakTestCase.java':[
482,
],
}
