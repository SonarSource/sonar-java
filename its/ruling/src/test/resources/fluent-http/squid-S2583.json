{
'net.code-story:http:src/main/java/net/codestory/http/misc/MemoizingSupplier.java':[
42,
],
}
