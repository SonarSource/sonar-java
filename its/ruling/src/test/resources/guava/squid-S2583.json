{
'com.google.guava:guava:src/com/google/common/base/Preconditions.java':[
211,
227,
249,
],
'com.google.guava:guava:src/com/google/common/collect/AbstractMapBasedMultiset.java':[
277,
287,
],
'com.google.guava:guava:src/com/google/common/collect/CollectPreconditions.java':[
30,
32,
],
'com.google.guava:guava:src/com/google/common/collect/ImmutableMultimap.java':[
215,
],
'com.google.guava:guava:src/com/google/common/collect/ObjectArrays.java':[
239,
],
'com.google.guava:guava:src/com/google/common/collect/Ordering.java':[
320,
322,
],
'com.google.guava:guava:src/com/google/common/collect/SortedMultisets.java':[
157,
],
'com.google.guava:guava:src/com/google/common/collect/StandardTable.java':[
589,
711,
],
'com.google.guava:guava:src/com/google/common/escape/Escapers.java':[
224,
],
}
