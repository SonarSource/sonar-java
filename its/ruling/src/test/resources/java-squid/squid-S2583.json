{
'org.sonarsource.java:java-squid:src/main/java/org/sonar/java/filters/SuppressWarningsFilter.java':[
63,
],
'org.sonarsource.java:java-squid:src/main/java/org/sonar/java/model/LiteralUtils.java':[
71,
],
'org.sonarsource.java:java-squid:src/main/java/org/sonar/java/model/VisitorsBridge.java':[
256,
267,
],
'org.sonarsource.java:java-squid:src/main/java/org/sonar/java/model/declaration/EnumConstantTreeImpl.java':[
71,
],
'org.sonarsource.java:java-squid:src/main/java/org/sonar/java/resolve/JavaSymbol.java':[
550,
],
'org.sonarsource.java:java-squid:src/main/java/org/sonar/java/resolve/Resolve.java':[
86,
],
'org.sonarsource.java:java-squid:src/main/java/org/sonar/java/syntaxtoken/FirstSyntaxTokenFinder.java':[
222,
252,
265,
],
}
