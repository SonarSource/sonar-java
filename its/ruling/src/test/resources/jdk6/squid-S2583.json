{
'jdk6:java/awt/ComponentOrientation.java':[
172,
],
'jdk6:java/awt/Container.java':[
4021,
],
'jdk6:java/awt/KeyboardFocusManager.java':[
214,
2226,
],
'jdk6:java/awt/MenuBar.java':[
168,
],
'jdk6:java/awt/color/ICC_Profile.java':[
1085,
],
'jdk6:java/awt/datatransfer/DataFlavor.java':[
301,
],
'jdk6:java/awt/dnd/DropTarget.java':[
365,
410,
],
'jdk6:java/awt/font/TextLayout.java':[
464,
],
'jdk6:java/awt/geom/CubicCurve2D.java':[
1227,
],
'jdk6:java/awt/geom/Path2D.java':[
1953,
1993,
],
'jdk6:java/awt/geom/QuadCurve2D.java':[
986,
],
'jdk6:java/awt/image/DataBufferUShort.java':[
120,
149,
],
'jdk6:java/awt/image/LookupOp.java':[
245,
],
'jdk6:java/awt/image/Raster.java':[
368,
],
'jdk6:java/beans/beancontext/BeanContextServicesSupport.java':[
446,
],
'jdk6:java/io/LineNumberInputStream.java':[
135,
],
'jdk6:java/net/InetAddress.java':[
1240,
],
'jdk6:java/net/Socket.java':[
374,
],
'jdk6:java/net/URI.java':[
3333,
],
'jdk6:java/net/URL.java':[
366,
],
'jdk6:java/nio/ByteBuffer.java':[
1104,
1131,
],
'jdk6:java/nio/CharBuffer.java':[
1104,
1131,
],
'jdk6:java/nio/DoubleBuffer.java':[
1104,
1131,
],
'jdk6:java/nio/FloatBuffer.java':[
1104,
1131,
],
'jdk6:java/nio/IntBuffer.java':[
1104,
1131,
],
'jdk6:java/nio/LongBuffer.java':[
1104,
1131,
],
'jdk6:java/nio/ShortBuffer.java':[
1104,
1131,
],
'jdk6:java/security/Identity.java':[
252,
],
'jdk6:java/security/SecureClassLoader.java':[
237,
],
'jdk6:java/security/Security.java':[
595,
],
'jdk6:java/sql/SQLException.java':[
56,
82,
105,
126,
150,
172,
197,
223,
],
'jdk6:java/text/PatternEntry.java':[
155,
],
'jdk6:java/util/ArrayList.java':[
572,
],
'jdk6:java/util/ResourceBundle.java':[
2332,
],
'jdk6:java/util/Scanner.java':[
996,
],
'jdk6:java/util/concurrent/CyclicBarrier.java':[
203,
217,
],
'jdk6:java/util/concurrent/ScheduledThreadPoolExecutor.java':[
236,
],
'jdk6:java/util/jar/JarFile.java':[
320,
],
'jdk6:java/util/logging/LogManager.java':[
499,
771,
],
}
