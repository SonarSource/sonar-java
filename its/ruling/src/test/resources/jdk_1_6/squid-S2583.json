{
'jdk_1_6:java/awt/Container.java':[
4021,
],
'jdk_1_6:java/awt/KeyboardFocusManager.java':[
2226,
],
'jdk_1_6:java/awt/MenuBar.java':[
168,
],
'jdk_1_6:java/awt/dnd/DropTarget.java':[
365,
410,
],
'jdk_1_6:java/awt/geom/QuadCurve2D.java':[
986,
],
'jdk_1_6:java/awt/image/LookupOp.java':[
245,
],
'jdk_1_6:java/awt/image/Raster.java':[
368,
],
'jdk_1_6:java/io/LineNumberInputStream.java':[
135,
],
'jdk_1_6:java/math/BigDecimal.java':[
3660,
],
'jdk_1_6:java/net/ServerSocket.java':[
470,
475,
],
'jdk_1_6:java/nio/ByteBuffer.java':[
1104,
1131,
],
'jdk_1_6:java/nio/CharBuffer.java':[
1104,
1131,
],
'jdk_1_6:java/nio/DoubleBuffer.java':[
1104,
1131,
],
'jdk_1_6:java/nio/FloatBuffer.java':[
1104,
1131,
],
'jdk_1_6:java/nio/IntBuffer.java':[
1104,
1131,
],
'jdk_1_6:java/nio/LongBuffer.java':[
1104,
1131,
],
'jdk_1_6:java/nio/ShortBuffer.java':[
1104,
1131,
],
'jdk_1_6:java/security/Identity.java':[
252,
],
'jdk_1_6:java/security/SecureClassLoader.java':[
237,
],
'jdk_1_6:java/text/PatternEntry.java':[
155,
],
'jdk_1_6:java/util/ArrayList.java':[
572,
],
'jdk_1_6:java/util/Scanner.java':[
996,
],
'jdk_1_6:java/util/concurrent/CyclicBarrier.java':[
203,
217,
],
'jdk_1_6:java/util/concurrent/ScheduledThreadPoolExecutor.java':[
236,
],
'jdk_1_6:java/util/concurrent/ThreadPoolExecutor.java':[
654,
874,
1349,
1461,
],
'jdk_1_6:java/util/logging/LogManager.java':[
771,
],
}
