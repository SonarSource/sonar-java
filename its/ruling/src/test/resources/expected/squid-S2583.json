{
'project:com/sun/corba/se/impl/encoding/CDROutputStream_1_0.java':[
1376,
],
'project:com/sun/corba/se/impl/naming/cosnaming/InterOperableNamingImpl.java':[
199,
],
'project:com/sun/imageio/plugins/bmp/BMPImageReader.java':[
384,
445,
],
'project:com/sun/jmx/snmp/IPAcl/Parser.java':[
70,
123,
126,
128,
155,
158,
160,
192,
256,
295,
298,
300,
349,
352,
354,
385,
414,
460,
550,
553,
555,
593,
596,
598,
617,
656,
659,
661,
698,
701,
703,
747,
750,
752,
779,
815,
818,
820,
846,
849,
851,
870,
909,
912,
914,
951,
954,
956,
],
'project:com/sun/jmx/snmp/daemon/SnmpInformRequest.java':[
823,
],
'project:com/sun/jmx/snmp/daemon/SnmpSocket.java':[
220,
231,
],
'project:com/sun/org/apache/regexp/internal/RECompiler.java':[
1091,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/NodeCounter.java':[
215,
],
'project:com/sun/org/apache/xerces/internal/dom/ObjectFactory.java':[
152,
],
'project:com/sun/org/apache/xerces/internal/impl/dtd/XMLDTDValidator.java':[
1533,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/ObjectFactory.java':[
152,
],
'project:com/sun/org/apache/xerces/internal/impl/xpath/XPath.java':[
416,
1350,
],
'project:com/sun/org/apache/xerces/internal/impl/xpath/regex/RangeToken.java':[
146,
158,
168,
299,
388,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/models/XSDFACM.java':[
1117,
],
'project:com/sun/org/apache/xerces/internal/parsers/ObjectFactory.java':[
150,
],
'project:com/sun/org/apache/xerces/internal/xinclude/ObjectFactory.java':[
150,
],
'project:com/sun/org/apache/xerces/internal/xpointer/XPointerHandler.java':[
701,
805,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMDefaultBaseTraversers.java':[
432,
],
'project:com/sun/org/apache/xml/internal/serialize/DOMSerializerImpl.java':[
1098,
],
'project:com/sun/org/apache/xml/internal/serialize/ObjectFactory.java':[
150,
],
'project:com/sun/org/apache/xml/internal/serializer/ToStream.java':[
1901,
],
'project:com/sun/org/apache/xml/internal/utils/NodeVector.java':[
479,
],
'project:com/sun/org/apache/xpath/internal/axes/ChildTestIterator.java':[
83,
],
'project:com/sun/org/apache/xpath/internal/compiler/Lexer.java':[
599,
],
'project:java/lang/invoke/ToGeneric.java':[
98,
],
'project:java/net/URI.java':[
3352,
],
'project:java/net/URLStreamHandler.java':[
148,
],
'project:java/text/DecimalFormat.java':[
2440,
],
'project:java/text/MessageFormat.java':[
1241,
],
'project:java/text/PatternEntry.java':[
172,
],
'project:java/util/ComparableTimSort.java':[
709,
828,
],
'project:java/util/TimSort.java':[
742,
861,
],
'project:java/util/concurrent/ForkJoinTask.java':[
263,
],
'project:java/util/logging/LogManager.java':[
274,
],
'project:java/util/regex/Pattern.java':[
1084,
],
'project:javax/management/ObjectName.java':[
463,
521,
598,
],
'project:javax/management/remote/rmi/RMIConnector.java':[
466,
],
'project:javax/security/auth/kerberos/ServicePermission.java':[
243,
],
'project:javax/swing/plaf/basic/BasicListUI.java':[
1194,
],
'project:javax/swing/plaf/basic/BasicSplitPaneUI.java':[
1342,
],
'project:javax/swing/plaf/metal/MetalIconFactory.java':[
1155,
],
'project:javax/swing/text/AbstractWriter.java':[
591,
635,
],
'project:javax/swing/text/Utilities.java':[
566,
],
'project:javax/swing/text/html/CSS.java':[
2601,
3264,
],
'project:javax/swing/text/html/CSSBorder.java':[
303,
],
'project:javax/swing/text/html/HTMLDocument.java':[
2355,
],
'project:javax/swing/text/html/StyleSheet.java':[
1559,
],
'project:javax/xml/datatype/FactoryFinder.java':[
197,
],
'project:javax/xml/parsers/FactoryFinder.java':[
197,
],
'project:javax/xml/stream/FactoryFinder.java':[
220,
],
'project:javax/xml/transform/FactoryFinder.java':[
197,
],
'project:javax/xml/validation/SchemaFactoryFinder.java':[
190,
],
'project:javax/xml/xpath/XPathFactoryFinder.java':[
187,
],
}
